paparazzi-gradle-plugin/src/test/projects/multiplatform-plugin-with-android/build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
```

Comment:
It would be great if you could add, let's say, a Compose view test here so like the other projects in this folder we guarantee that it's playing well with the new changes to support KMP.

Reply:
agreed.  I'll merge this and let's follow up with a Compose KMP test.